docs/reference/esql/esql-across-clusters.asciidoc

```diff
@@ -19,6 +20,70 @@ include::{es-ref-dir}/search/search-your-data/search-across-clusters.asciidoc[ta
 
 include::{es-ref-dir}/search/search-your-data/search-across-clusters.asciidoc[tag=ccs-proxy-mode]
 
+[discrete]
+[[esql-ccs-security-model]]
+==== Security model
+
+{es} supports two security models for cross-cluster search (CCS):
+
+* <<esql-ccs-security-model-certificate, TLS certificate authentication>>
+* <<esql-ccs-security-model-api-key, API key authentication>>
+
+[discrete]
+[[esql-ccs-security-model-certificate]]
+===== TLS certificate authentication
+
+TLS certificate authentication makes sense in single administrator scenarios where you have full control over both clusters.
+Importantly, you must ensure that named roles and their privileges are identical in both clusters.
+Refer to <<remote-clusters-cert, TLS certificate authentication>> for prerequisites and detailed setup instructions.
+
+[discrete]
+[[esql-ccs-security-model-api-key]]
+===== API key authentication
+
+[NOTE]
+====
+`ENRICH` is *not supported* in this version when using {esql} with the API key based security model.
+====
+
+The following information pertains to using {esql} across clusters with the <<remote-clusters-api-key, *API key based security model*>>.
+API key based cross-cluster search (CCS) enables more granular control over allowed actions between clusters.
+As a result, defining roles and privileges is more complex than with the certificate based security model.
```

Comment:
Can basically remove this sentence and lean on previous one, if we don't want my non-specific signpost that API route is more complex :)

Reply:
Just trying to avoid calling one more complex than the other. Keeping roles in sync is also complex, just in a different way. Users may want to adopt the API key based model due to the complexities of mirroring roles (we don't provide any tooling to do that).